### PR TITLE
logrotate container intended to be used by node fluentds

### PR DESCRIPTION
**Pull request by coffeepac**
this container will delete old logs and the directories of removed
pods.  for more context, please see ADA-62

### CODE REVIEW

File: load-and-sleep.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+#  copy the script logrotate-run.sh to the 15 minute crontab entry
+cp run-logrotate.sh /etc/periodic/15min/
+if [ $? -ne 0 ]; then
+  echo "copy command failed"
+  exit 1
+fi
+
+#  sleep forever
+while true; do
```
Comment (davidewatson) [inline load-and-sleep.sh]: This is fine, but if the version of alpine linux this container is based on supports ```sleep infinity``` that would be less hackish. There is also code from the grc.io/pause container, but that seems like too much work for what should be a simple task. 100 years is effectively forever, right? :)

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM alpine
+
+RUN apk update && apk add logrotate
+
+COPY load-and-sleep.sh /load-and-sleep.sh
```
Comment (davidewatson) [inline Dockerfile]: Maybe make this a single COPY command, or not if you think it is more readable this way. https://docs.docker.com/engine/reference/builder/#copy

File: load-and-sleep.sh (hunk 1 above)
Comment (coffeepac) [inline load-and-sleep.sh]: I checked that and alpine uses busybox for a lot of system utils so:  no dice.

File: Dockerfile (hunk 2 above)
Comment (coffeepac) [inline Dockerfile]: changes made.  new revision added.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,21 @@
...
+# entrypoint
+the script `load-and-sleep.sh` will load logrotate's execution script into
+the 15 minute periodic run folder.  the execution script is 
+`/run-logrotate.sh` and contains a call to execute logrotate 
+with the below config file.  
+
+# config
+the config file is landed at `/logrotate.conf`.  If you wish to override it,
+land a new config to the same spot.  the config assumes all log files are 
+being landed at `/var/log/apps/*` where the final wildcard is the pod unique
```
Comment (davidewatson) [inline README.md]: "logrotate does not have a recursive option, so currently this will go two levels deep looking for logs"

Comment (coffeepac) [inline README.md]: logrotate treats '*' the same as '**'.

**Comment by coffeepac** [comment]
Open for awhile and no comments in 20 days.  Merging.

**Pull request merged**